micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/config/PropertyAbsentOrEnabledCondition.groovy

```diff
@@ -0,0 +1,31 @@
+package com.ofg.infrastructure.config
+
+import groovy.transform.CompileStatic
+import org.apache.commons.lang.BooleanUtils
+import org.springframework.context.annotation.Condition
+import org.springframework.context.annotation.ConditionContext
+import org.springframework.core.type.AnnotatedTypeMetadata
+
+@CompileStatic
+abstract class PropertyAbsentOrEnabledCondition implements Condition {
```

Comment:
Very good idea to extract such class, actually useful even outside our project. Spring Boot PR maybe?